### PR TITLE
Update jsonFieldMappings metatype.properties to reflect field omissio…

### PR DESCRIPTION
**Pull request by ellen-lau**
#build

### CODE REVIEW

**Comment by ellen-lau** [comment]
#build

**Comment by ellen-lau** [comment]
fixes: #10431

File: dev/com.ibm.ws.logging.osgi/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -91,4 +91,4 @@ console.format.simple=Use the simple logging format.
...
-json.fields.desc=When logs are in JSON format, use this attribute to replace default field names with new field names. Configure the new field name by using the following format: defaultFieldName:newFieldName. For field names that are associated with logs of a specified source, use the following format: [source:]?defaultFieldName:newFieldName, where [source] is the source you want to specify (such as message, trace, or accessLog). To rename multiple fields, specify a comma-separated list of field name mappings.
\ No newline at end of file
+json.fields.desc=When logs are in JSON format, use this attribute to replace default field names with new field names. Use this attribute to omit fields as well. Configure the new field name by using the following format: defaultFieldName:newFieldName?. For field names that are associated with logs of a specified source, use the following format: [source:]?defaultFieldName:newFieldName?, where [source] is the source you want to specify (such as message, trace, or accessLog). Leave newFieldNameblank if you want to omit the field. To rename/omit multiple fields, specify a comma-separated list of field name mappings.
```
Comment (dmuelle) [inline dev/com.ibm.ws.logging.osgi/resources/OSGI-INF/l10n/metatype.properties:94]: `Use this attribute to omit fields` - does this mean omit the filed from the logs?

` Leave newFieldNameblank if you want to omit the field`

Does this mean you would use use the following format?
`defaultFieldName:`

**Comment by skasund** [comment]
L2 message review not required

File: dev/com.ibm.ws.logging.osgi/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
Comment (ellen-lau) [reply dev/com.ibm.ws.logging.osgi/resources/OSGI-INF/l10n/metatype.properties:94]: Hi @dmuelle,

1. `Use this attribute to omit fields` - yes, the field will be omitted from the logs

2. `Leave newFieldNameblank if you want to omit the field`

Does this mean you would use use the following format?
`defaultFieldName:`

Yes that's correct, that is the format being used.

Comment (dmuelle) [inline dev/com.ibm.ws.logging.osgi/resources/OSGI-INF/l10n/metatype.properties:94]: suggested edit-

When logs are in JSON format, use this attribute to replace default field names with new field names or to omit fields from the logs. To replace a field name, configure the new field name by using the following format: defaultFieldName:newFieldName?. For field names that are associated with logs of a specified source, use the following format: [source:]?defaultFieldName:newFieldName?, where [source] is the source you want to specify, such as message, trace, or accessLog. To omit a field from the logs, specify the field name without a replacement, as shown in the following example: defaultFieldName:. To rename or omit multiple fields, specify a comma-separated list of field name mappings.

**Comment by ellen-lau** [comment]
#build

**Comment by ellen-lau** [comment]
#run-libby-bot

**Comment by ellen-lau** [comment]
#build

**Comment by ellen-lau** [comment]
#build

**Comment by ellen-lau** [comment]
#build

**Pull request merged**